deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -293,6 +305,15 @@ bool QmlStreamer::Impl::_setupRootItem()
              [this]() { _updateSizes( QSize( _rootItem->width(),
                                              _rootItem->height( ))); } );
 
+    connect( _quickWindow, &QQuickWindow::minimumWidthChanged,
```

Comment:
It saves the private function declaration. Even liking lambdas, I would not have thought of that in this case.